### ISSUE TITLE
[BUG] Persona 3 FES - Broken Minimap

### ISSUE THREAD

**Issue by PawTM** [PCSX2/pcsx2#4069]
**Describe the bug**
In Persona 3 FES, the individual parts of the Dungeon minimap get misaligned from each other.
![grafik](https://user-images.githubusercontent.com/62476228/103414235-69071880-4b7d-11eb-93fc-a571598c3c4c.png)

**To Reproduce**
Go into any ingame Dungeon, e.g. Tartarus, and uncover part of the minimap.

**Expected behavior**
On a PS2, the individual parts are aligned.
![grafik](https://user-images.githubusercontent.com/62476228/103414377-fb0f2100-4b7d-11eb-8474-7b22882d2ce6.png)

**GS Settings**
This issue happens on any resolution etc. , even SW Mode and Native, but can be fixed by setting the Texture Offset Upscaling Hack to 500|500.

**System Info (please complete the following information):**
 - PCSX2 Revision: 1.7 dev-817
 - OS: Windows 10
 - CPU: i5-8400
 - GPU: GTX 1060

**Logs and Dumps**
[P3FES-GSDump.zip](https://github.com/PCSX2/pcsx2/files/5757497/P3Fes.zip)

**Comment by LOuroboros** [comment]
This is still happening on the latest versions of the emulator, for whatever that's worth.

**Comment by F0bes** [comment]
Even on software mode?

**Comment by refractionpcsx2** [comment]
yeah I think this is one of those jank games (like Dark Cloud 2) where how it handles sprites is really jank lol

**Comment by LOuroboros** [comment]
> Even on software mode?

Yup.

![pcsx2-qtx64_20220627_2026251](https://user-images.githubusercontent.com/4485172/176053435-67e20efa-f285-4a30-9425-09e62107be01.png)

**Comment by KawaiiDinosaur** [comment]
Same results here using v1.7.3069. 
Setting the Texture Offset Upscaling hack to 500x500 fixes it, as previously mentioned. But the issue remains if you use the software renderer.

![gs_20220711011821_Shin Megami Tensei - Persona 3 FES_SLUS-21621](https://user-images.githubusercontent.com/36821201/178399921-218e03fa-0bf9-4063-bb14-4835da63ea91.png)

![gs_20220711232542_Shin Megami Tensei - Persona 3 FES_SLUS-21621](https://user-images.githubusercontent.com/36821201/178399922-69b9ce74-b82b-4667-b925-aaf9d96e1fed.png)

**Comment by refractionpcsx2** [comment]
The minimap is due to how the STQ coordinates are handled for the textures for the minimap squares, I have noted it her https://github.com/PCSX2/pcsx2/pull/6553#issuecomment-1173177908

I could fix it, but I don't think the fix for just that is necessarily right, since Manhunt has a similar issue, but it's handled slightly differently due to precision issues, so we need to get that right.

**Comment by Mrlinkwii** [comment]
this is fixed on SW renderer as of  v1.7.5236
![image](https://github.com/PCSX2/pcsx2/assets/17685135/84e8927d-3ad7-4d3c-a912-c57edd6a737e)
